ansible/vtestbed.csv

```diff
@@ -4,3 +4,4 @@ vms-kvm-t0-64,vms6-1,t0-64,docker-ptf,ptf-01,10.250.0.102/24,fec0::ffff:afa:2/64
 vms-kvm-t1-lag,vms6-2,t1-lag,docker-ptf,ptf-02,10.250.0.106/24,fec0::ffff:afa:6/64,server_1,VM0104,[vlab-03],Tests virtual switch vm
 vms-kvm-t0-2,vms6-3,t0,docker-ptf,ptf-03,10.250.0.108/24,fec0::ffff:afa:8/64,server_1,VM0104,[vlab-04],Tests virtual switch vm
 vms-kvm-dual-t0,vms6-4,dualtor,docker-ptf,ptf-04,10.250.0.109/24,fec0::ffff:afa:9/64,server_1,VM0108,[vlab-05;vlab-06],Dual-TOR testbed
+vms-kvm-multi-asic-t1-lag,vms6-4,t1-lag,docker-ptf,ptf-unknown,10.250.0.106/24,fec0::ffff:afa:6/64,server_1,VM0104,[vlab-07],Tests multi-asic virtual switch vm
```

Comment:
I think a new PTF should be defined for this multi-ASIC setup.
* `ptf-05` 
* `10.250.0.109`
* `fec0::ffff:afa:a/64`

The `ptf-05` host also should be added to `ansible/veos_vtb`.

Reply:
Added new ptf-docker as suggested.